Makefile

```diff
@@ -59,6 +59,10 @@ test-coupled:    ## Test if the coupling with MAgPIE works. Takes significantly
 	$(info Coupling tests take around 75 minutes to run, please be patient)
 	@R_PROFILE_USER= TESTTHAT_RUN_SLOW=TRUE Rscript -e 'testthat::test_file("tests/testthat/test_20-coupled.R")'
 
+test-coupled-slurm: ## test-coupled, but on slurm
```

Comment:
Shouldn't test-coupled be removed from make commands so you can only run coupled test with SLURM?

Reply:
I don't think so. The model runs are on slurm anyway, the difference is whether also the test itself is on slurm. I think normally you should not do that (because it reserves a full CPU basically for waiting), but sometimes if the internet connection is unstable (train…) it is a helpful option because then the tests do not fail just because your console lost connection.